distribution/debian/packages.bzl

```diff
@@ -0,0 +1,102 @@
+load("@rules_pkg//pkg:pkg.bzl", "pkg_tar")
+load("@rules_pkg//pkg:deb.bzl", "pkg_deb")
+
+GLIBC_MIN_VERSION = "2.27"
+
+def envoy_pkg_deb(
+        name,
+        data,
+        homepage = "https://www.envoyproxy.io/",
+        description = "Envoy built for Debian/Ubuntu",
+        preinst = "//distribution/debian:preinst",
+        postinst = "//distribution/debian:postinst",
+        supported_distributions = "buster bullseye bionic focal hirstute impish",
```

Comment:
```suggestion
        supported_distributions = "buster bullseye bionic focal hirsute impish",
```

Reply:
Though actually hirsute is already EOL: https://endoflife.date/ubuntu

Reply:
hi, thanks for headsup ill remove it